postgres/datadog_checks/postgres/postgres.py

```diff
@@ -460,7 +481,7 @@ def _collect_wal_metrics(self):
 
     def _get_local_wal_file_age(self):
         wal_log_dir = os.path.join(self._config.data_directory, "pg_xlog")
-        if not os.path.isdir(wal_log_dir):
+        if not self._config.data_directory or not os.path.isdir(wal_log_dir):
```

Comment:
Is this new conditional necessary? We already access this variable in the line above to build `wal_log_dir` and then check that its a directory